packages/kuma-gui/src/app/vue/index.ts

```diff
@@ -148,6 +178,22 @@ export const services = (app: Record<string, Token>): ServiceDefinition[] => {
         $.navigationGuards,
       ],
     }],
+    [token('application.directives'), {
+      service: () => {
+        return []
+      },
+      labels: [
+        $.directives,
+      ],
+    }],
+    [token('application.componentWalkers'), {
+      service: () => {
+        return []
+      },
+      labels: [
+        $.componentWalkers,
```

Comment:
Naming of these is getting a bit uncomfortable 😬 we already have things called `walkers` for `routeWalkers`, so I used the same naming for now.